src/main/java/com/cloudbees/dockerpublish/DockerBuilder.java

```diff
@@ -332,7 +332,7 @@ public boolean isApplicable(Class<? extends AbstractProject> aClass) {
          * This human readable name is used in the configuration screen.
          */
         public String getDisplayName() {
-            return "Docker build and publish";
+            return "CloudBees Docker Build and Publish";
```

Comment:
Do we really want this to be the build step name ? I mean, will all plugin author prepend UI labels with his company name ?